### ISSUE TITLE
HKSV mp4 video files consumed entire raspberry SD card space

### ISSUE THREAD

**Issue by bondskin** [koush/scrypted#462]
This morning my Scrypted instance did, root file system was filled up to 100%

Reason:
huge amount of mp4 files (18GB)  at
~/.scrypted/volume/plugins/@scrypted/homekit/files/hksv

Deleting the files cured the symptom.

I never did this behaviour before, running Scrypted with months now.
Could this have been caused by a recent plugin update?

**Comment by bondskin** [comment]
after deleting files, it fills up rapidly again

> -rw-r--r-- 1 pi pi 13157384 Dec 11 11:40 13-1670755238643.mp4
> -rw-r--r-- 1 pi pi    88826 Dec 11 11:41 13-1670755259569.jpg
> -rw-r--r-- 1 pi pi       83 Dec 11 11:41 13-1670755259569.json
> -rw-r--r-- 1 pi pi 12163459 Dec 11 11:41 13-1670755259569.mp4
> -rw-r--r-- 1 pi pi       83 Dec 11 11:41 13-1670755278915.json
> -rw-r--r-- 1 pi pi 10803492 Dec 11 11:41 13-1670755278915.mp4
> -rw------- 1 pi pi       86 Dec 11 11:41 nohup.out
> [1]+  Exit 2                  nohup python3 ./speedtest2influx.py
> pi@raspberrypi:~/.scrypted/volume/plugins/@scrypted/homekit/files/hksv $ du -h
> 187M    .
> pi@raspberrypi:~/.scrypted/volume/plugins/@scrypted/homekit/files/hksv $ du -h
> 201M    .
> pi@raspberrypi:~/.scrypted/volume/plugins/@scrypted/homekit/files/hksv $

**Comment by koush** [comment]
Yeah turn that off. I’ll be removing thst feature or making it harder to find cause people get confused and think it’s required. It’s a debug tool.
